modules/post/multi/gather/aws_ec2_instance_metadata.rb

```diff
@@ -50,14 +50,14 @@ def check_curl
 
   def get_aws_metadata(base_uri, base_resp)
     r = {}
-    base_resp.split(/\r\n/).each do |l|
-      new_uri = base_uri.merge("./#{l}")
-      if l =~ %r{/$}
+    base_resp.split(/\n/).each do |l|
```

Comment:
Is there a reason that `.split(/\r\n/)` was replaced with `.split(/\n/)` ? This is kind of important.

If it was causing problems, consider using `/\r?\n/` instead.